featurebyte/routes/deployment/api.py

```diff
@@ -72,13 +72,16 @@ async def get_deployment(request: Request, deployment_id: PydanticObjectId) -> D
 
 @router.patch("/{deployment_id}")
 async def update_deployment(
-    request: Request, deployment_id: PydanticObjectId, data: DeploymentUpdate
+    request: Request, deployment_id: PydanticObjectId, data: DeploymentUpdate, response: Response
 ) -> Optional[Task]:
     """
     Update Deployment
     """
     controller = request.state.app_container.deployment_controller
     task: Task = await controller.update_deployment(document_id=deployment_id, data=data)
+    if isinstance(task, Task):
+        # if task is returned, it means the deployment is being updated asynchronously
+        response.status_code = HTTPStatus.ACCEPTED
     return task
```

Comment:
Update this part so that it is aligned with the main branch to return 202 when an async job is created.